G/Git/Compat.toml

```diff
@@ -5,5 +5,10 @@ VersionParsing = "1"
 julia = ["0.7", "1"]
 
 [1]
-Git_jll = "2.31.0-2"
 julia = "1.6.0-1"
+
+["1-1.2"]
+Git_jll = "2.31.0-2"
+
```

Comment:
Maybe this should be made incompatible with 2.36, to prevent https://github.com/JuliaPackaging/Yggdrasil/pull/4987#issuecomment-1436490385

Reply:
Done

Reply:
Thanks!